src/scenes/start_scene.py

```diff
@@ -71,19 +71,19 @@ def __init__(self, screen: pygame.Surface) -> None:
         self.level: Optional[LevelScene] = None
         self.exit: QuitActionKind = QuitActionKind.CONTINUE
 
-        StartScene.load_options()
+        self.options_file = None
+        self.load_options()
```

Comment:
I think you need to rebase your branch.

Reply:
I'll start from scratch and just change a reasonable number of scripts at a time. I think I started with level_scene.py, and the dependencies spiraled quicker than I expected.